lib/Backend/Interpreter.hs

```diff
@@ -0,0 +1,18 @@
+-- SPDX-FileCopyrightText: 2022 Serokell <https://serokell.io>
+--
+-- SPDX-License-Identifier: MPL-2.0
+
+module Backend.Interpreter where
+
+import Polysemy
+
+import Backend (Backend (..), BackendEffect (..), SomeBackend (..), Effects)
+
+runBackendIO :: Effects r => Sem (BackendEffect ': r) a -> Sem r a
```

Comment:
This function doesn't deal with `IO`, so I think it should be renamed to just `runBackend`.

Similarly, the `runBackendIO_` in `Main` could  be renamed to simply `runBackendIO`.

Reply:
Done

Reply:
I think you forgot this bit:

> Similarly, the runBackendIO_ in Main could be renamed to simply runBackendIO.

Reply:
Done